pkgs/gnomod/gnomod.go

```diff
@@ -0,0 +1,210 @@
+package gnomod
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/gnolang/gno/pkgs/gnolang"
+	"github.com/gnolang/gno/pkgs/std"
+	"golang.org/x/mod/modfile"
+	"golang.org/x/mod/module"
+)
+
+var (
+	source        = "test3.gno.land:36657" // "127.0.0.1:26657"
```

Comment:
this needs to be dynamic (can be done in a next PR)

Reply:
marked for future PRs